x-pack/metricbeat/module/gcp/timeseries_metadata_collector.go

```diff
@@ -51,23 +51,25 @@ func (s *StackdriverTimeSeriesMetadataCollector) Metadata(ctx context.Context, i
 		accountID = in.Resource.Labels[TimeSeriesResponsePathForECSAccountID]
 	}
 
-	ecs := mapstr.M{
-		ECSCloud: mapstr.M{
-			ECSCloudAccount: mapstr.M{
-				ECSCloudAccountID:   accountID,
-				ECSCloudAccountName: accountID,
-			},
-			ECSCloudProvider: "gcp",
+	ecsCloud := mapstr.M{
+		ECSCloudAccount: mapstr.M{
+			ECSCloudAccountID:   accountID,
+			ECSCloudAccountName: accountID,
 		},
+		ECSCloudProvider: "gcp",
```

Comment:
whats the change here for to introduce ecsCloud?

Reply:
Oh this is not related to the metadata, it was a bug in ecs cloud fields, I can revert the commit since it's not related to this issue.

Reply:
oh gotcha! we can keep it here! thanks!